models/source/im-bm-ci/im_roasted_format.sql

```diff
@@ -0,0 +1,19 @@
+{{
+  config({
+    "materialized" : "table",
+    "post-hook" : [
+      "grant select on table {{this}} to group non_gl_read_only"
+      ]
+    })
+}}
+
+select
+    udf_roasted_format_id::int as roasted_format_id,
+    udf_format_desc as format_desc,
+    nullif(udf_roasted_lbs, '')::float as roasted_lbs,
+    nullif(udf_servings, '')::int as servings,
+    nullif(udf_case_eq, '')::float as case_eq,
+    nullif(udf_subunit_format, '')::int as sub_unit_format,
```

Comment:
Rename to sub_unit_format_id, and move this field to below the format_desc to match the organization of the other models